### PR TITLE
Security | Updating users, build.env and sso policy

### PR DESCRIPTION
**Pull request by exequielrafaela**
## What?
* Updating users permissions

## Why?
* Least privilege and necessary access.

### CODE REVIEW

**Review by diego-ojeda-binbash — APPROVED** [review]

File: build.env [hunk 1]
```diff
@@ -5,4 +5,4 @@ PROJECT=bb
 MFA_ENABLED=false
 
 # Terraform
-TERRAFORM_IMAGE_TAG=1.3.5-0.1.17
+TERRAFORM_IMAGE_TAG=1.3.5-0.2.0
```
Comment (diego-ojeda-binbash) [inline build.env:8]: @exequielrafaela Isn't that image only compatible with RC `1.12.5rc1`?

Comment (exequielrafaela) [reply build.env:8]: @diego-ojeda-binbash You're right. I'll discard this change for the moment. Thanks!

**Review by rodriguez-matias — APPROVED** [review]
LGTM

File: build.env (hunk 1 above)
Comment (exequielrafaela) [reply build.env:8]: @diego-ojeda-binbash the latest release covers this. I'll move forward and merge it now.

**Pull request merged**